### PR TITLE
Add nationality/ethnicity to patient lookup

### LINKED ISSUE #3911: Nationality/Ethnicity should be saved from patient lookups
## Is your feature request related to a problem? Please describe.

The patient lookup isn't mapping nationality and ethnicities

## Describe the solution you'd like

Add the fields

## Implementation

TBD

## Describe alternatives you've considered

TBD

## Additional context

TBD

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3911 

## Change summary

- Saves ethnicity/nationality of a patient when looking up

## Testing

- [ ] Lookup a patient with an ethnicity and/or nationality - it should show on step 2 of vaccine dispensing

### Related areas to think about

N/A

### CODE REVIEW

**Review by mark-prins — APPROVED** [review]
Great! that does it.
Though I've realised in testing that the Tonga server needs updating.. it isn't returning either ID when you lookup.

File: src/sync/lookupApiUtils.js [hunk 1]
```diff
@@ -210,6 +210,8 @@ export const processPatientResponse = response => {
       date_of_birth,
       nameInsuranceJoin,
       nameNotes,
+      nationality_ID,
+      ethnicity_ID,
```
Comment (mark-prins) [inline src/sync/lookupApiUtils.js:214]: @joshxg we don't have `female` either. Would you be able to add that in to this PR? or should I make a new issue just for that..?

Comment (joshxg) [reply src/sync/lookupApiUtils.js:215]: nice! 
a1b3bf4

**Comment by joshxg** [comment]
re-request as I did a commit and the branch is protected so the review got dismissed

**Review by mark-prins — APPROVED** [review]
🎉

**Pull request merged**